packages/components/src/components/text/text.jsx

```diff
@@ -1,27 +1,27 @@
 import classNames from 'classnames';
 import React from 'react';
 
-const Text = ({ children, size, color, align, weight, lineHeight, as, className }) => {
+const Text = ({ children, ...props }) => {
```

Comment:
Could we explicitly list the props here for clarity?

Reply:
At first it was like this, but I changed it as per @reza-fs comment on this PR.

Reply:
You can still have an `...otherProps` argument, but it would be good if the default expected props were there, e.g.:

```JS
const Text = ({ children, align, as, className, color, line_height, size, weight, ...otherProps  }) => {
    // component definition.
}
```

You can then pass `otherProps` into the returned components.